### PR TITLE
Ratification Polls 10 10 22

### PR DESCRIPTION
**Pull request by Hipogeo**
@prose11 @blimpa

### CODE REVIEW

File: governance/polls/Ratification Poll - Offboarding the Strategic Happiness Core Unit - SH-001 (MIP39c3-SP3) - October 10, 2022.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, October 10 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c3SP3 being accepted and implemented in the Maker Protocol.
```
Comment (Gala-G) [inline governance/polls/Ratification Poll - Offboarding the Strategic Happiness Core Unit - SH-001 (MIP39c3-SP3) - October 10, 2022.md:38]: Change to MIP39c3-SP3

File: governance/polls/Ratification Poll - Offboarding the Events Core Unit - EVENTS-001 (MIP39c3-SP4) - October 10, 2022.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c3-SP4 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MakerDAO is restructuring the work and the workforce across the DAO according to the Endgame Plan. The mandate for this Core Unit does not support a standalone core unit and will be better aligned within a cluster of a MetaDAO. As a result, this core unit must be offboarded to support the transition.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/b1f879da42b30df08497e95641a1b6291bc79879/MIP39/MIP39c3-Subproposals/MIP39c3-SP4.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Offboarding the Events Core Unit - EVENTS-001 (MIP39c3-SP4) - October 10, 2022.md:45]:
```suggestion
* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/0e92ffa48c584e67cdd2d31166f7d0625855203a/MIP39/MIP39c3-Subproposals/MIP39c3-SP4.md)
```

Comment (blimpa) [reply governance/polls/Ratification Poll - Offboarding the Events Core Unit - EVENTS-001 (MIP39c3-SP4) - October 10, 2022.md:45]: Status change

**Review by Gala-G — CHANGES_REQUESTED** [review]

File: governance/polls/Ratification Poll - Facilitator Offboarding - RWF-001 (MIP41c5-SP12) - October 10, 2022.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This subproposal relieves the current RWF-001 Facilitator, @williamr, from his responsibilities as a mandated actor and leader of the RWF-001 Core Unit. The outgoing Facilitator will not be considered a member of the RWF-001 Core Unit unless retained by the Interim or a future Facilitator.*
+
+*All unspent funds and assets should immediately be handed over to an appointed Interim Facilitator.*
+
+If this proposal passes and [MIP39c3-SP5](https://forum.makerdao.com/t/mip39c3-sp5-offboarding-the-real-world-finance-core-unit-rwf-001/17753) passes at the same time, the other one should take precedence over and supersede this one.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/46ac501e11beaaef232d881d7706fc7505de52f2/MIP41/MIP41c5-Subproposals/MIP41c5-SP12.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Facilitator Offboarding - RWF-001 (MIP41c5-SP12) - October 10, 2022.md:49]:
```suggestion
* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/dbc71ae46d7e1e2248984d1461f6d826acd719f9/MIP41/MIP41c5-Subproposals/MIP41c5-SP12.md)
```

Comment (blimpa) [reply governance/polls/Ratification Poll - Facilitator Offboarding - RWF-001 (MIP41c5-SP12) - October 10, 2022.md:49]: Status change

File: governance/polls/Ratification Poll - Offboarding the Real-world Finance Core Unit - RWF-001 (MIP39c3-SP5) - October 10, 2022.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c3-SP5 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MakerDAO is restructuring the work and the workforce across the DAO according to the Endgame Plan. The mandate for the Real-world Finance Core Unit will be better aligned within a cluster of a MetaDAO. As a result, this Core Unit must be offboarded to support the transition.*
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Offboarding the Real-world Finance Core Unit - RWF-001 (MIP39c3-SP5) - October 10, 2022.md:42]:
```suggestion
*"MakerDAO is restructuring the work and the workforce across the DAO according to the Endgame Plan. The mandate for the Real-world Finance Core Unit will be better aligned within a cluster of a MetaDAO. As a result, this Core Unit must be offboarded to support the transition."*
```

File: governance/polls/Ratification Poll - Offboarding the Real-world Finance Core Unit - RWF-001 (MIP39c3-SP5) - October 10, 2022.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/506b5e49bc94a228624f6c4a8e83b7004d38e12d/MIP39/MIP39c3-Subproposals/MIP39c3-SP5.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Offboarding the Real-world Finance Core Unit - RWF-001 (MIP39c3-SP5) - October 10, 2022.md:45]:
```suggestion
* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/55e70dc7ef318498574737faa4370a14471e267d/MIP39/MIP39c3-Subproposals/MIP39c3-SP5.md)
```

File: governance/polls/Ratification Poll - Offboarding the Strategic Happiness Core Unit - SH-001 (MIP39c3-SP3) - October 10, 2022.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MakerDAO is restructuring the work and the workforce across the DAO according to the Endgame Plan. The mandate for this Core Unit does not support a standalone Core Unit and will be better aligned within a cluster of a MetaDAO. As a result, this Core Unit will be offboarded to support the transition.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/2069d46fce904563b5665a871b81eefa78e06dbf/MIP39/MIP39c3-Subproposals/MIP39c3-SP3.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Offboarding the Strategic Happiness Core Unit - SH-001 (MIP39c3-SP3) - October 10, 2022.md:45]:
```suggestion
* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/d098ea16627c32ae28de97738b32c1cdf960fcea/MIP39/MIP39c3-Subproposals/MIP39c3-SP3.md)
```

File: governance/polls/Ratification Poll - Offboarding the Strategic Happiness Core Unit - SH-001 (MIP39c3-SP3) - October 10, 2022.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP39c3SP3 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MakerDAO is restructuring the work and the workforce across the DAO according to the Endgame Plan. The mandate for this Core Unit does not support a standalone Core Unit and will be better aligned within a cluster of a MetaDAO. As a result, this Core Unit will be offboarded to support the transition.*
```
Comment (blimpa) [inline governance/polls/Ratification Poll - Offboarding the Strategic Happiness Core Unit - SH-001 (MIP39c3-SP3) - October 10, 2022.md:42]:
```suggestion
*"MakerDAO is restructuring the work and the workforce across the DAO according to the Endgame Plan. The mandate for this Core Unit does not support a standalone Core Unit and will be better aligned within a cluster of a MetaDAO. As a result, this Core Unit will be offboarded to support the transition."*
```

**Review by blimpa — CHANGES_REQUESTED** [review]

**Pull request merged**